ApiTests/Utils/XpathPatterns.cs

```diff
@@ -9,5 +9,6 @@ namespace TestProject.Utils
     internal class XpathPatterns
     {
         public static readonly string preciseTextXpath = "//*[text()='{0}']";
+        public static readonly string preciseInputXpath = "//input[@type='text']";
```

Comment:
this class is supposed to be used to store locators patterns, not specific implementations. I'd suggest moving this lovator to the class where it's used, as we won't be able to reuse this locator for elements from other pages (unlike the one above)